packages/transaction-pay-controller/src/strategy/relay/relay-quotes.ts

```diff
@@ -289,13 +288,48 @@ function getFeatureFlags(messenger: TransactionPayControllerMessenger) {
     'RemoteFeatureFlagController:getState',
   );
 
-  const featureFlags = featureFlagState.remoteFeatureFlags?.confirmation_pay as
-    | Record<string, string>
-    | undefined;
+  const featureFlags = featureFlagState.remoteFeatureFlags
+    ?.confirmations_pay as Record<string, string> | undefined;
 
   const relayQuoteUrl = featureFlags?.relayQuoteUrl ?? RELAY_URL_QUOTE;
 
   return {
     relayQuoteUrl,
   };
 }
+
+/**
+ * Calculates source network cost from a Relay quote.
+ *
+ * @param quote - Relay quote.
+ * @param messenger - Controller messenger.
+ * @returns Total source network cost in USD and fiat.
```

Comment:
what is the difference between USD and fiat? Is fiat the user's selected currency?